plugins/SegmentEditor/tests/UI/SegmentSelectorEditor_spec.js

```diff
@@ -99,10 +86,12 @@ describe("SegmentSelectorEditorTest", function () {
     });
 
     it("should add an OR condition when clicking on add OR", async function() {
-        await page.click('.segmentEditorPanel .segment-add-or');
+        await page.evaluate(function() {
+            $('.segmentEditorPanel .segment-add-or').click();
+        });
```

Comment:
Did you try using the suggested blur instead?
`await page.$eval('.segmentEditorPanel .segmentRow0 .ui-autocomplete-input', e => e.blur());`

That way this UI test shouldn't show the autosuggest box, which is unexpected.